.github/workflows/e2e-tests.yml

```diff
@@ -116,10 +117,12 @@ jobs:
           .github/files/e2e-tests/map-plugins-for-e2e-env.sh
           echo "::endgroup::"
         else
-          echo "::group::Build plugin(s)"
-          cd $PROJECT_PATH
-          pnpm run build
-          echo "::endgroup::"
+          if [ "${RUN_NAME}" != atomic ]; then
```

Comment:
Probably no need for this additional nesting. This conditional could be combined as `elif [ "${RUN_NAME}" != atomic ]; then`